### ISSUE TITLE
Developer Guide uses deprecated code

### ISSUE THREAD

**Issue by sebthom** [jkuhnert/ognl#114]
The example in the https://github.com/jkuhnert/ognl/blob/master/docs/DeveloperGuide.md#embedding-ognl uses deprecated code and should be updated:

```java
public Object getValue(OgnlContext context, Object rootObject) throws OgnlException {
   return Ognl.getValue(getExpression(), context, rootObject); // <-- deprecated
}
```

What is a non-deprecated alternative that resembles the same functionality?

**Comment by lukaszlenart** [comment]
It was deprecated because it was using `addDefaultContext()` without providing implementation of `MemberAccess`, but I added a default implementation sometime ago and now the deprecation doesn't make sense. Fixed that!

**Comment by sebthom** [comment]
@lukaszlenart I think you forgot to de-deprecate the `Ognl.getValue` methods in the code.

**Comment by lukaszlenart** [comment]
Good catch, fixed! OGNL 3.2.18 is under way!

**Comment by sebthom** [comment]
@lukaszlenart thanks!!!
